src/Microsoft.DotNet.Interactive/Commands/RequestInput.cs

```diff
@@ -3,16 +3,25 @@
 
 namespace Microsoft.DotNet.Interactive.Commands;
 
+public enum InputType
```

Comment:
We shouldn't introduce an enum for this.

Reply:
Would you prefer a new input token for connection strings e.g. "@connectionString:myConnectionString", or to use the prompt part after the existing input token e.g. "@input:connectionString"?

Reply:
Would you prefer a new token for connection strings e.g. "@ connectionString:myConnectionString", or to use the prompt part after the existing input token e.g. "@ input:connectionString"?

Reply:
The goal is for `@input:x` to be able to work in different frontends, whether a REPL, automation, VS Code, Jupyter, or ADS. ADS currently has a specific experience for connection strings but this should be able to fall back gracefully rather than block people in other environments. 

With that in mind, we also need a way to differentiate between a text input prompt and a connection chooser in ADS. This seems more like we need a way to hint at a preference rather than say `@input` and `@connectionString` are fundamentally different.

Reply:
Switched to using "@input:connectionString"